packages/gatsby-theme-carbon/src/components/ImageCard/image-card.scss

```diff
@@ -101,6 +99,15 @@
   transition: opacity $duration--fast-01;
 }
 
+.#{$prefix}--image-card__spacing {
```

Comment:
I feel like this is just as likely to randomly stop working as the original solution haha. If the magic 1px switcheroo is the only solution that's fine, just wanna make sure we've explored more solid options.